tests/test_plotting.py

```diff
@@ -36,6 +36,17 @@ def test_plot_frequency_recency_matrix(self):
 
         plt.show()
 
+    def test_plot_probability_alive_matrix(self):
+        print plt.matplotlib.__version__
```

Comment:
Whoops, I didn't want that print to survive at all. I've changed it to the import that I was actually going for -- so that it can be run as a standalone test, and to be consistent with the other tests.